src/core/CCArray.ml

```diff
@@ -197,8 +197,21 @@ let rev a =
   rev [| |] = [| |]
 *)
 
+exception Found
+
+let mem ?(eq = Stdlib.(=)) elt a =
+  try
+    Array.iter (fun e -> if eq e elt then raise Found) a;
```

Comment:
`raise_notrace` can be slightly better here

Reply:
Done